packages/indexer-common/src/indexer-management/__tests__/actions.ts

```diff
@@ -245,7 +245,7 @@ const setup = async () => {
   })
   networkSubgraph = await NetworkSubgraph.create({
     logger,
-    endpoint: 'https://gateway.testnet.thegraph.com/network',
+    endpoint: 'https://gateway.thegraph.com/network',
```

Comment:
I've updated all these to use hosted-service endpoints recently; let's do that here too?  If you rebase on main you should see it come through.